### ISSUE TITLE
ranger-search-files does not work

### ISSUE THREAD

**Issue by psamim** [syl20bnr/spacemacs#5029]
#### Description
In default ranger `f` is bound to `ranger-search-files` while in ranger layer in Spacemacs this key is shadowed by the default evil-mode behaviour of `f` key. I suggest using different key binding for `ranger-search-files`.

#### Reproduction guide
- Start Emacs
- Be sure to use `vim` editing mode
- Be sure ranger layer is loaded
- Open ranger with `SPC a r`
- Press `f`

*Observed behaviour:*
I observe Emacs is waiting for the next letter, because `f<X>` puts the cursor on the next occurrence of the letter <X>.

*Expected behaviour:*
I expect it brings `ranger-search-files` as it is the default ranger behaviour.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.9
- Spacemacs branch: nil (rev. 0562f05)
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-help-tooltip t)
 better-defaults emacs-lisp javascript clojure haskell python lua c-c++ html shell-scripts ess sql go scala java react git markdown org restclient selectric erc dockerfile latex
 (evil-snipe :variables evil-snipe-enable-alternate-f-and-t-behaviors t)
 evil-commentary spell-checking syntax-checking version-control ranger semicolon org-latex-classes org-psamim psamim-defaults)
```

**Comment by StreakyCobra** [comment]
It works as expected here. Can you update your issue to follow [this template](https://github.com/syl20bnr/spacemacs/wiki/Debugging#how-to-make-a-great-bug-report)?

**Comment by psamim** [comment]
@StreakyCobra Sure, I updated the issue.

**Comment by StreakyCobra** [comment]
Can you try to remove `evil-snipe` from you layer list and try again?

**Comment by psamim** [comment]
I did and now it is working as expected. Should we add a hook to disable evil-snipe in ranger?

**Comment by StreakyCobra** [comment]
Yes probably. @ralesi any opinion on this?

**Comment by ralesi** [comment]
@streakycobra That can work, I will investigate how evil-snipe hooks into evil-mode bindings.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
